src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -157,6 +163,15 @@ private void ComputeParameters()
 
             var diagnostics = DiagnosticBag.GetInstance();
             SyntaxToken arglistToken;
+
+            foreach (var param in _syntax.ParameterList.Parameters)
+            {
+                foreach (var attrList in param.AttributeLists)
```

Comment:
The parameter symbol itself is binding the attributes and they go into the bag which is passed to the constructor.

In the second PR, this "strict" symbol is removed entirely and the attribute forcing code is moved into this method.